qa/util/src/main/java/io/camunda/qa/util/cluster/TestStandaloneCamunda.java

```diff
@@ -69,37 +64,41 @@ public final class TestStandaloneCamunda extends TestSpringApplication<TestStand
           .withEnv("action.destructive_requires_name", "false");
   private final BrokerBasedProperties brokerProperties;
   private final OperateProperties operateProperties;
+  private final TasklistProperties tasklistProperties;
 
   public TestStandaloneCamunda() {
     super(
-        BrokerModuleConfiguration.class,
+        CommonsModuleConfiguration.class,
         OperateModuleConfiguration.class,
+        TasklistModuleConfiguration.class,
         WebappsModuleConfiguration.class,
-        CommonsModuleConfiguration.class,
-        // test overrides - to control data clean up; (and some components are not installed on
-        // Tests)
-        TestElasticsearchSchemaManager.class,
-        TestSchemaStartup.class);
+        BrokerModuleConfiguration.class,
+        GatewayModuleConfiguration.class,
```

Comment:
❓ Was this necessary? 🤔 It was working without before.

Reply:
The BrokerModule yes, otherwise Tasklist is not able to retrieve the partitions in correct way. 
The GatewayModuleConfiguration not, removed this one.